### PR TITLE
HBS out, Svelte In

### PR DESCRIPTION
**Pull request by PuruVJ**

### CODE REVIEW

**Comment by PuruVJ** [comment]
Please no one kill me if things break.

File: twindle-cli/src/renderer/epub/render-template.js [hunk 1]
```diff
@@ -62,7 +55,7 @@ async function renderTwitterTemplate(data) {
   const optionDetails = {
     title: authorNames + "'s Thread",
     author: authorNames,
-    html: threadContent,
+    html,
     css,
```
Comment (tolgaerdonmez) [inline twindle-cli/src/renderer/epub/render-template.js:59]: this line should be deleted

Comment (tolgaerdonmez) [reply twindle-cli/src/renderer/epub/render-template.js]: the `css`

Comment (PuruVJ) [reply twindle-cli/src/renderer/epub/render-template.js:59]: Which one? 59?

**Review by tolgaerdonmez — CHANGES_REQUESTED** [review]
That line in epub renderer should be deleted, and all is well :))
Nice work!

**Comment by Mira-Alf** [comment]
We will make sure Twitter works. Then svelte templates need to be built for github, hackernews and readability - I can work on it but we need to organize code for separating the templates for the sources.

**Comment by PuruVJ** [comment]
> We will make sure Twitter works. Then svelte templates need to be built for github, hackernews and readability - I can work on it but we need to organize code for separating the templates for the sources.

When it gets merged, we need to really test it manually. Loads of things might have been broken up. PDF as well as EPUB.

This project is getting too big now 😅

**Comment by Mira-Alf** [comment]
Well good luck to us then!

**Pull request merged**